src/actions/favorites.js

```diff
@@ -0,0 +1,18 @@
+export const ADD_FAVORITE = 'ADD_FAVORITE';
+export const REMOVE_FAVORITE = 'REMOVE_FAVORITE';
+export const TOGGLE_FAVORITE = 'TOGGLE_FAVORITE';
+
+export const addFavorite = (gif) => ({
+  type: ADD_FAVORITE,
+  payload: gif
```

Comment:
As a note, it is a good practice to wrap the payload inside an object (this convention was first created by the `Flux` team, and it's recommend to use also in `Redux` for consistency).

```js
payload: { gif },
```